bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/editor/auxfile/AuxFileEditor.scala

```diff
@@ -27,7 +27,7 @@ object AuxFileEditor {
       lab.text = size.toString
   })
 
-  private val Format = new SimpleDateFormat("MM/dd/yy HH:mm:ss") {
+  private val Format = new SimpleDateFormat("yyyy-MMM-dd HH:mm:ss") {
```

Comment:
I probably mislead you talking about "dialog boxes".  It's just the "File Attachment" tab on the top-level program node.  Those table columns are evenly distributed based on available space.  We can always explicitly shrink the "Size" and "NGO Check?" column if necessary.